code/game/machinery/doors/airlock.dm

```diff
@@ -1240,6 +1240,32 @@ GLOBAL_LIST_EMPTY(airlock_emissive_underlays)
 				to_chat(user, "<span class='warning'>Despite your attempts, [src] refuses to open.</span>")
 
 /obj/machinery/door/airlock/open(forced=0)
+	var/list/sound_pool = list(
```

Comment:
this will probably get called a lot so it probably doesn't hurt to micro-optimize this a bit
```suggestion
	var/static/list/sound_pool = list(
```